src/index.js

```diff
@@ -1,32 +1,40 @@
+import {
+  add, editDescription, load, remove, save,
+} from './add-and-remove';
 import './style.css';
 
-const todo = [
-  {
-    description: 'List structure.',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Interactive list.',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Add and remove items.',
-    completed: false,
-    index: 2,
-  },
-];
+const todo = load() || [];
```

Comment:
- [ ] Finally, the index of the elements should start with `1` in todo data, but in yours, it's starting from `0`. You can set the started number to 1 👍